src/library/trackset/crate/cratestorage.cpp

```diff
@@ -357,13 +323,9 @@ CrateSelectResult CrateStorage::selectCratesByIds(
     DEBUG_ASSERT(!subselectForCrateIds.isEmpty());
 
     FwdSqlQuery query(m_database,
-            mixxx::DbConnection::collateLexicographically(QString(
-                    "SELECT * FROM %1 WHERE %2 %3 (%4) ORDER BY %5").arg(
-                            CRATE_TABLE,
-                            CRATETABLE_ID,
-                            subselectPrefix,
-                            subselectForCrateIds,
-                            CRATETABLE_NAME)));
+            mixxx::DbConnection::collateLexicographically(
+                    QString("SELECT * FROM %1 WHERE %2 %3 (%4) ORDER BY %5")
+                            .arg(CRATE_TABLE, CRATETABLE_ID, subselectPrefix, subselectForCrateIds, CRATETABLE_NAME)));
```

Comment:
long line

Reply:
This was intentional - my rationale was to only wrap the arguments is the format String is wrapped as well.

Reply:
Anyways, fixed it, wrapped the format string as well